geoindicators/src/main/groovy/org/orbisgis/BlockIndicators.groovy

```diff
@@ -21,6 +19,11 @@ import org.orbisgis.processmanagerapi.IProcess
  * @author Jérémy Bernard
  */
 static IProcess holeAreaDensity() {
+    //Definition of constant values
+    def final GEOMETRIC_FIELD = "the_geom"
```

Comment:
Does that make sense to put it out the closure ?

Reply:
Making them as constants ensure that those variable won't be changed and it's also a good practice to extract constants.

Reply:
Not sure to be aligned with this argument ;-)